tests/utils/exec/cluster.go

```diff
@@ -31,6 +31,8 @@ type ClusterCreationArgs struct {
 	DefaultMPLabels                      map[string]string `json:"default_mp_labels,omitempty"`
 	DisableSCPChecks                     bool              `json:"disable_scp_checks,omitempty"`
 	MultiAZ                              bool              `json:"multi_az,omitempty"`
+	Properties                           map[string]string `json:"properties,omitempty"`
+	CustomProperties                     map[string]string `json:"custom_properties,omitempty"`
```

Comment:
Why Properties and another CustomProperties? Let's keep one only. Is properties used anywhere?

Reply:
you are right, fixed :)